README.md

```diff
@@ -1,10 +1,19 @@
+![NorESM logo](doc/img/NORESM-logo.png)
+
 # Norwegian Earth System Model (NorESM)
 
+The Norwegian Earth System Model NorESM has been developed since 2007 and has been an important tool for Norwegian climate researchers in the study of the past, present and future climate. NorESM has also contributed to climate simulation that has been used in research assessed in the IPCC’s fifth main report.
```

Comment:
maybe add brackets around NorESM e.g. The Norwegian Earth System Model (NorESM)...

Reply:
Done :)